test/missing_semicolon.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
+    <script>
+        var xmlHttp = new XMLHttpRequest();
```

Comment:
Same as [this comment](https://github.com/js-cookie/js-cookie/pull/171/files#r59530280)